client/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "devDependencies": {
     "@hot-loader/react-dom": "~16.13",
-    "webpack-dev-server": "~3.11"
+    "webpack-dev-server": "^1.16.5"
```

Comment:
this will likely have a profound impact on various configuration elements of your application. Is there a solid reason for this version change